geoengine/datasets.py

```diff
@@ -61,12 +61,13 @@ def to_dict(self) -> Dict[str, str]:
 class CustomOgrSourceTimeFormat(OgrSourceTimeFormat):
     '''A custom OGR time format'''
 
-    custom: str
+    # related: https://github.com/python/mypy/issues/12975
```

Comment:
Why is this comment here? I don't understand it.

Reply:
The mypy error with the original code is as follows:

> Incompatible types in assignment (expression has type "str", base class "OgrSourceTimeFormat" defined the type as "Callable[[Type[OgrSourceTimeFormat], s mypy [65, 5]

Somehow mypy messes up if the field name of the subclass and the method name of the superclass are equal. Apparently this is not yet supported by mypy and therefore the name of the field has to be changed.
If changing the name is ok, I can remove the reference link, otherwise we would have to investigate further on this.

Reply:
Yes, then just remove the comment, please.

Reply:
Gone with [52f3cec](https://github.com/geo-engine/geoengine-python/pull/70/commits/52f3cec883e67a03700efe7f8dc4db0beaf55494)